source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -505,6 +550,107 @@ This option allows to disable the Audit health check during the Whodata engine s
 .. warning:: The health check ensures that the rules required by Whodata can be set in Audit correctly and also that the generated events can be obtained. Disabling the health check may cause functioning problems in Whodata and loss of FIM events.
 
 
+process_priority
+^^^^^^^^^^^^^^^^
+
+.. versionadded:: 3.12.0
+
+Set the nice value for Syscheck process.
+
++--------------------+------------------------------------+
+| **Default value**  | 10                                 |
++--------------------+------------------------------------+
+| **Allowed values** | Integer number between -20 and 19. |
++--------------------+------------------------------------+
+
+The "niceness" scale in Linux goes from -20 to 19, whereas -20 is the highest priority and 19 the lowest priority.
+
+For Windows the scale is translated as described in the following table:
+
++------------+------------------------------+
+| -20 to -10 | THREAD_PRIORITY_HIGHEST      |
++------------+------------------------------+
+| -9 to -5   | THREAD_PRIORITY_ABOVE_NORMAL |
++------------+------------------------------+
+| -4 to 0    | THREAD_PRIORITY_NORMAL       |
++------------+------------------------------+
+| 1 to 5     | THREAD_PRIORITY_BELOW_NORMAL |
++------------+------------------------------+
+| 6 to 10    | THREAD_PRIORITY_LOWEST       |
++------------+------------------------------+
+| 11 to 19   | THREAD_PRIORITY_IDLE         |
++------------+------------------------------+
+
+
+max_eps
+^^^^^^^
+
+.. versionadded:: 3.12.0
+
+Set the maximum output throughput.
+
++--------------------+---------------------------------------+
+| **Default value**  | 200                                   |
++--------------------+---------------------------------------+
+| **Allowed values** | Integer number between 1 and 1000000. |
++--------------------+---------------------------------------+
+
+
+inventory
+^^^^^^^^^
+
+.. versionadded:: 3.12.0
+
+The database synchronization settings will be configured inside this tag.
+
+.. code-block:: xml
+
+    <!-- Database synchronization settings -->
+    <inventory>
+      <sync_interval>5m</sync_interval>
+      <response_timeout>30</response_timeout>
+      <sync_queue_size>16384</sync_queue_size>
+    </inventory>
+
+
+**sync_interval**
+
+.. versionadded:: 3.12.0
+
+Specifies the number of seconds between every inventory synchronization.
+
++--------------------+---------------------------------------+
+| **Default value**  | 300                                   |
++--------------------+---------------------------------------+
+| **Allowed values** | Any number greater than or equal to 0.|
++--------------------+---------------------------------------+
+
+**response_timeout**
+
+.. versionadded:: 3.12.0
+
+Specifies the time elapsed in seconds since the agent sends the message to the manager and recives the response.
```

Comment:
`recives` → `receives`

I think we should note that a timeout does not mean that the agent will start a new synchronization, but marks the message as unanswered, and the agent may start a new synchronization session (at the defined interval).

In other words, the agent will not start a new synchronization session until the last message that it delivered to the manager is answered or timed-out.